From d812abf567398cd8d527e519b8d917a2280eac82 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:37:38 -0400
Subject: [alsa] trident: Remove unused variable in trident_main.c

Message-id: <140972823573-370-patchset-jkysela@redhat.com>
Patchwork-id: 91066
O-Subject: [RHEL 7.1 PATCH 369/442] ALSA: trident: Remove unused variable in trident_main.c
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

'private_data' is not used in the function. Remove it.

Signed-off-by: Sachin Kamat <sachin.kamat@samsung.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 8d9048643f32d795fd6dbf71d9dd5469611151cd)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/trident/trident_main.c b/sound/pci/trident/trident_main.c
index 1272c18..da875dc 100644
--- a/sound/pci/trident/trident_main.c
+++ b/sound/pci/trident/trident_main.c
@@ -3880,14 +3880,12 @@ void snd_trident_free_voice(struct snd_trident * trident, struct snd_trident_voi
 {
  unsigned long flags;
  void (*private_free)(struct snd_trident_voice *);
- void *private_data;
 
  if (voice == NULL || !voice->use)
   return;
  snd_trident_clear_voices(trident, voice->number, voice->number);
  spin_lock_irqsave(&trident->voice_alloc, flags);
  private_free = voice->private_free;
- private_data = voice->private_data;
  voice->private_free = NULL;
  voice->private_data = NULL;
  if (voice->pcm)
-- 
1.7.1